config/condition_no.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+// NoConditionConfig is used to set a non-null value to a task's condition
+// configuration block when it is unconfigured.
+type NoConditionConfig struct{}
+
+// Copy returns a deep copy of this configuration.
+func (c *NoConditionConfig) Copy() MonitorConfig {
+	if c == nil {
+		return nil
+	}
+	return &NoConditionConfig{}
+}
+
+// Merge combines all values in this configuration with the values in the other
+// configuration, with values in the other configuration taking precedence.
+// Maps and slices are merged, most other values are overwritten. Complex
+// structs define their own merge functionality.
+func (c *NoConditionConfig) Merge(o MonitorConfig) MonitorConfig {
+	if c == nil {
+		return nil
+	}
+	return &NoConditionConfig{}
+}
+
+// Finalize ensures there no nil pointers.
+func (c *NoConditionConfig) Finalize(services []string) {
+}
+
+// Validate validates the values and required options. This method is recommended
+// to run after Finalize() to ensure the configuration is safe to proceed.
+func (c *NoConditionConfig) Validate() error {
+	return nil
+}
+
+// GoString defines the printable version of this struct.
+func (c *NoConditionConfig) GoString() string {
+	return ""
+}
```

Comment:
If we leave `""` then condition and source input defaults print like this: `Condition:SourceInput:` seems a bit weird, maybe `"&Condition{&NoCondition{}}"` or print empty braces `{}`?

summary: return `"{}"` or `"&Condition{&NoCondition{}}"`

Reply:
Good point, I like the suggestion to return the empty braces. Thanks!